### DISCUSSION TITLE
are examples up to date?

### DISCUSSION THREAD

**Discussion by bubo-py** [hajimehoshi/ebiten#1478] [Q&A] [answered]
I tried to run a couple of examples in my local directory, however, in every of them:
```   
        "github.com/hajimehoshi/ebiten/v2"
	"github.com/hajimehoshi/ebiten/v2/audio"
	"github.com/hajimehoshi/ebiten/v2/ebitenutil"
```
the "/v2" seems already out of use. These examples are available on the website or here in repo, so the question is if they are up to date with the current project structure.

**Comment by hajimehoshi** [comment, marked as answer]
> the "/v2" seems already out of use. 

They are not out of use. For this import paths rule, see https://github.com/golang/go/wiki/Modules#semantic-import-versioning
